weather_test.go

```diff
@@ -35,7 +36,12 @@ func TestGetSendsCorrectURL(t *testing.T) {
 	t.Parallel()
 	var w weather.Conditions
 	s := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Println("Inside the handler func")
+		data, err := ioutil.ReadFile("testdata/weather.json")
+		if err != nil {
+			fmt.Println("Error: ", err)
```

Comment:
That's fine, but since this is a test, we could fail the test instead, couldn't we?